.storybook/recipes/NumberIconList/NumberIconList.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+
+import { NumberIconList, Props } from './NumberIconList';
+import { NumberIcon } from '../../../src';
+
+export default {
+  title: 'Recipes/NumberIconList',
+  component: NumberIconList,
+} as Meta;
+
+const Template: Story<Props> = (args) => (
+  <NumberIconList>
+    <NumberIcon aria-label="Item 1" number={1} size="sm" />
+    <NumberIcon
+      aria-label="Item 2"
+      incomplete={true}
+      number={2}
+      numberIconTitle={`incomplete step 2`}
+      size="sm"
+    />
+    <NumberIcon aria-label="Item 3" number={3} size="sm" />
+    <NumberIcon
+      aria-label="Item 4"
+      incomplete={true}
+      number={4}
+      numberIconTitle={`incomplete step 4`}
+      size="sm"
+    />
+    <NumberIcon
+      aria-label="Item 5"
+      incomplete={true}
+      number={5}
+      numberIconTitle={`incomplete step 5`}
+      size="sm"
+    />
+    <NumberIcon
+      aria-label="Item 6"
+      incomplete={true}
+      number={6}
+      numberIconTitle={`incomplete step 6`}
+      size="sm"
+    />
+  </NumberIconList>
+);
+
+export const Default = Template.bind({});
```

Comment:
Think with #1129 we want the stories written in csf v3?

Reply:
@dierat How do we want to handle this since it seems like that hasn't been merged yet?

Reply:
It's safe to change it here; that's just for new files generated after it merges.